tests/XRootD/cache.cfg

```diff
@@ -0,0 +1,25 @@
+set name = cache
+set port = 7096
+
+set src = $SOURCE_DIR
+set pwd = $PWD
+
+xrootd.trace all
+pfc.trace info
+
+xrootd.seclib libXrdSec.so
+xrd.protocol XrdHttp:$port libXrdHttp.so
+
+http.desthttps false
+http.selfhttps2http false
+
+xrootd.chksum max 2 md5 adler32 crc32 crc32c
+
+ofs.osslib libXrdPss.so
+#ofs.ckslib * libXrdPss.so
```

Comment:
Why add a commented out line?

Reply:
I can drop it for now.  I had wanted to enable checksums but the server was failing to start -- we can keep it simpler and revisit later why it's crashing.